.travis.yml

```diff
@@ -0,0 +1,10 @@
+language: java
+jdk:
+  - oraclejdk7
+  - openjdk7
+
+before_install:
+ - chmod +x gradlew
```

Comment:
Ups, i copied from another travis file.

But... i will remove if you want, but i think it's not a bad idea to make it executable, what if it changed somewhen?